mllib/src/main/scala/org/apache/spark/mllib/feature/HashingTF.scala

```diff
@@ -135,18 +135,20 @@ object HashingTF {
 
   private[spark] val Murmur3: String = "murmur3"
 
-  private[spark] val supportedHashAlgorithms: Array[String] = Array(Native, Murmur3)
+  private[spark] val supportedHashAlgorithms: Array[String] = Array(Murmur3)
```

Comment:
So its a bit weird that we're using supportedHashAlgorithms in mllib from ml, maybe we should just move this over there to reduce the confusion? Since we still intend to (at least temporarily) support native in MLLib but not in ml.